extensions/fetch/26_fetch.js

```diff
@@ -269,9 +270,13 @@
     if (nullBodyStatus(response.status)) {
       core.close(resp.responseRid);
     } else {
-      response.body = new InnerBody(
-        createResponseBodyStream(resp.responseRid, terminator),
-      );
+      if (req.method === "HEAD" || req.method === "OPTIONS") {
+        response.body = null;
```

Comment:
```suggestion
        response.body = null;
        core.close(resp.responseRid);
```

Otherwise you will leak the response resource.